components/mitsubishi_uart/mitsubishi_uart.cpp

```diff
@@ -107,6 +107,7 @@ be about `update_interval` late from their actual time.  Generally the update in
 (default is 5seconds) this won't pose a practical problem.
 */
 void MitsubishiUART::update() {
+  this->_updateLoopCounter += 1;
```

Comment:
Why only every n-loops / 60 seconds? Is this a particularly expensive call to make?